### PR TITLE
ASSIGNMENT-6 DONE

### PR DESCRIPTION
**Pull request by nikhil1-successive**

### CODE REVIEW

File: Assignment-6/Ques3.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// 3.1 Create a class Person with properties name, age, gender, and interests. Add a method greeting() that returns a string introducing the person.
+// Also add a method farewell() that returns a string saying goodbye to the person.
+class Person1 {
```
Comment (shahbaz-successive) [inline Assignment-6/Ques3.js:3]: In all question use the same Person Class instead of creating a new one for each part

**Review by shahbaz-successive — CHANGES_REQUESTED** [review]
Please do feedback changes

File: Assignment-6/Ques3.js (hunk 1 above)
Comment (nikhil1-successive) [reply Assignment-6/Ques3.js]: Done

**Review by shahbaz-successive — APPROVED** [review]
👍

**Pull request merged**